src/TemplateEditor.js

```diff
@@ -61,10 +61,12 @@ class TemplateEditor extends React.Component {
     super(props);
 
     this.quill = React.createRef();
+    this.quillId = `template-editor-${new Date().getTime()}`;
```

Comment:
Perhaps generating a random UUID would be better?

Reply:
it would be, but i didn't want to add a dependency and current approach do the job

but I'm willing to use it instead of required.

Reply:
`uuid` is already a dependency of many, many other UI modules, so while I appreciate the concern, I think @MikeTaylor's suggestion is worthwhile. Thanks for this fix!